node-bb/README.md

```diff
@@ -0,0 +1,169 @@
+
+#### Upgrading To A Newer NodeBB Version
+
+Upgrading to a newer version uses the following process:
+
+1. Change the version of the From image in the Dockerfile to the newer version of NodeBB.
+1. Update the install/package.json to use packages from the newer versions package.json source.
+1. Replace the nodebb-plugins in the install/package.json with the current list of plugins used for TripleA Forums.
+1. Run Docker build to create the new image.
+
+#### Plugins  
+
+NodeBB customizations are plugins which are installed either through the admin interface or by adding them 
+to package.json directly. For the TripleA Forums, they are installed through the package.json.
+
+#### Installing/Upgrading A Plugin
+
+Installing or grading a plugin use the following process:
+
+1. Update the install/package.json to use the newer plugin.
+1. Run Docker build to create the new image.
+
+##### List of currently installed plugins. 
+
+NOTE: *Refer to install/package.json for the most up to date list.*
+
+```
+"nodebb-plugin-2factor": "^2.7.2",
+"nodebb-plugin-cards": "0.2.2",
+"nodebb-plugin-composer-default": "6.3.25",
+"nodebb-plugin-custom-pages": "^1.1.3",
+"nodebb-plugin-dbsearch": "4.0.7",
+"nodebb-plugin-desktop-notifications": "^0.3.3",
+"nodebb-plugin-emoji": "^3.3.0",
+"nodebb-plugin-emoji-android": "2.0.0",
+"nodebb-plugin-emoji-one": "^2.0.0",
+"nodebb-plugin-featured-threads": "0.2.1",
+"nodebb-plugin-github-embed": "^0.7.5",
+"nodebb-plugin-gravatar": "^2.2.4",
+"nodebb-plugin-markdown": "8.11.2",
+"nodebb-plugin-mentions": "2.7.4",
+"nodebb-plugin-newsletter": "^0.8.1",
+"nodebb-plugin-ns-embed": "^4.0.0",
+"nodebb-plugin-ns-login": "^3.0.0",
+"nodebb-plugin-poll": "^0.3.3",
+"nodebb-plugin-shoutbox": "^0.3.4",
+"nodebb-plugin-soundpack-default": "1.0.0",
+"nodebb-plugin-spam-be-gone": "0.6.7",
+"nodebb-plugin-sso-facebook": "^3.5.1",
+"nodebb-plugin-sso-github2": "2.0.2",
+"nodebb-plugin-sso-google": "^2.4.1",
+"nodebb-plugin-sso-twitter": "^2.5.8"
+```    
+
+#### Configuration
+
+The NodeBB configuration is a JSON file, ```config.json```, which is kept at the root of the NodeBB install.
+An example of that file can be found at ```example/config.json```. Information on all the configuration 
+options for NodeBB can be found [here](https://docs.nodebb.org/configuring/config/).
+
+Copy the example/config.json file to a location of your choice. Edit the file to update the parameters for the database. 
+
+**Key points of the configuration to remember**:
+
+1. The *url* parameter is referencing the address NodeBB is binding to inside the Docker container. This is different 
+from the address of the host machine.
+1. The *port* parameter is referencing the port NodeBB is listening on inside the Docker container. This does not 
+have to be the same port on the host machine. Docker maps the host machine port to the container port.  
+1. Database (Mongo) parameters *host* and *port* are referring to the address and port of the machine 
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `3`, was `1`](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6342386)